Gruntfile.js

```diff
@@ -148,7 +149,8 @@ module.exports = function (grunt) {
 
         sass: {
             options: {
-                implementation: sass
+                implementation: sass,
+                fiber: Fiber,
```

Comment:
As per https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/535 -- Fibers should keep the build fast